### PR TITLE
refactor(front): bye bye as

### PR DESCRIPTION
**Pull request by UnbearableBear**
the "as" tag is not necessary anymore !

### CODE REVIEW

File: packages/code-du-travail-frontend/pages/outils/convention-collective.js [hunk 1]
```diff
@@ -1,3 +1,28 @@
-import SearchConvention from "../convention-collective";
+import { Container, PageTitle, Section, Wrapper } from "@socialgouv/cdtn-ui";
```
Comment (UnbearableBear) [inline packages/code-du-travail-frontend/pages/outils/convention-collective.js:1]: I also moved the CC search page here once and for all.

**Comment by lionelB** [comment]
this will work only in nextjs 10 https://nextjs.org/blog/next-10#automatic-resolving-of-href

**Comment by UnbearableBear** [comment]
Docs say it works since 9.5.3 ? https://nextjs.org/docs/api-reference/next/link

`as: Before Next.js 9.5.3 this was used for dynamic routes, check our previous docs to see how it worked`

File: packages/code-du-travail-frontend/pages/outils/convention-collective.js (hunk 1 above)
Comment (UnbearableBear) [reply packages/code-du-travail-frontend/pages/outils/convention-collective.js:1]: (the production bug was due to a wrong "as" prop on Link component)

**Review by lionelB — APPROVED** [review]
looks good, need to see why E2E tests failed

**Comment by UnbearableBear** [comment]
> looks good, need to see why E2E tests failed

wrong url for convention collective page 😅

**Pull request merged**